lnwallet/channel.go

```diff
@@ -566,7 +566,7 @@ func (c *commitment) toDiskCommit(
 // restore commitment state written to disk back into memory once we need to
 // restart a channel session.
 func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
-	commitHeight uint64, htlc *channeldb.HTLC, localCommitKeys,
+	htlc *channeldb.HTLC, localCommitKeys *CommitmentKeyRing,
 	remoteCommitKeys *CommitmentKeyRing, whoseCommit lntypes.ChannelParty,
```

Comment:
Unnecessary `*CommitmentKeyRing` after `localCommitKeys`, same for `extractPayDescs`

Reply:
feelz

<img width="454" alt="omsac" src="https://github.com/user-attachments/assets/22d4676e-66ac-4f60-9cf1-26919d75318c">